doc/release-notes-6118.md

```diff
@@ -0,0 +1,5 @@
+Breaking Changes
+----------------
+
+* The Dash Core repository (`github.com/dashpay/dash`) will be using `develop` as its default branch. New clones
```

Comment:
nit: `https://github.com/dashpay/dash` or `github.com:dashpay/dash` ?

Reply:
Used the HTTPS URL as it's the most recognizable while omitting anything that doesn't strictly convey details about the path (the `https://`)